storage/cache/subtree_cache.go

```diff
@@ -58,6 +58,8 @@ const (
 	depthQuantum = 8
 	// logStrataDepth is the strata that must be used for all log subtrees.
 	logStrataDepth = 8
+	// maxLogDepth is the number of bits in a log path.
```

Comment:
Going past 64 bit depths would be a major upheaval, which is not likely to happen. But we should remove the need for users to hardcode the magic number '64'.